### DISCUSSION TITLE
show persian character

### DISCUSSION THREAD

**Discussion by Amirkhaledian** [dompdf/dompdf#2716] [Project Development]
I want to get a pdf file output, but in the output that contains Persian and English letters instead of Persian letters, the sign? is showing. Where does the problem come from and how should I fix it?

**Comment by bsweeney** [comment]
Please read through the [Unicode How-To](https://github.com/dompdf/dompdf/wiki/UnicodeHowTo), which details the steps you need to take. Also, to get more specific help you should share a sample of your HTML + CSS as well the PHP utilizing Dompdf.
